emacs/emacs.personal/custom-functions.el.symlink

```diff
@@ -1,3 +1,10 @@
+(defun show-and-copy-file-path-with-line-number ()
+  (interactive)
+  (let ((path (concat (buffer-file-name) ":" (number-to-string (line-number-at-pos)))))
+    (progn
```

Comment:
You don't need `progn` for this as all expressions after the binding list are evaluated in order.